x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -61,4 +62,6 @@ export function initServicesApi(core: CoreSetup) {
       return getService(serviceName, setup).catch(defaultErrorHandler);
     }
   });
+
+  ensureIndexPatternExists(server);
```

Comment:
Just curious: why is this inside the `initServicesApi` function?

Reply:
Since it's supposed to create the index on server startup, it's being called while the plugin initializes.

Reply:
Cool, we talked about this on a call but to re-cap: I think it makes sense to call this from inside the new Plugin file where the init api calls are made, and pass in `core` which has `core.server` on it, rather than calling it from within one of the existing init calls.